tests/bootstrap.php

```diff
@@ -5,15 +5,6 @@
 define('TESTS_TEMP_DIR', __DIR__ . '/temp');
```

Comment:
Is this file even necessary? Environmental variables can be set in `phpunit.xml`